resources/views/profile/two-factor-authentication-form.blade.php

```diff
@@ -142,69 +144,76 @@ class="w-full button-secondary sm:w-auto"
                     @lang('fortify::actions.recovery_codes')
                 </button>
 
-                <button type="submit" class="w-full button-primary sm:w-auto" wire:click="disableTwoFactorAuthentication">
+                <button
+                    type="submit"
+                    class="w-full button-primary sm:w-auto"
+                    wire:click="disableTwoFactorAuthentication"
+                    dusk="disable-one-time-password"
```

Comment:
```suggestion
                    dusk="disable-two-factor-authentication"
```

OTP is the password itself, you enable and disable 2FA with it.